runtime/js/99_main.js

```diff
@@ -536,10 +536,6 @@ function dispatchUnloadEvent() {
 }
 
 let hasBootstrapped = false;
-// Delete the `console` object that V8 automatically adds onto the global wrapper
-// object on context creation. We don't want this console object to shadow the
-// `console` object exposed by the ext/node globalThis proxy.
-delete globalThis.console;
```

Comment:
Why this is gone now?

Reply:
this was done so that `globalThis.console` (the v8 one), would not shadow `denoGlobals.console`. that is now not necessary anymore because we overwrite `globalThis.console` directly